### PR TITLE
Video Player Application

### LINKED ISSUE #88: Fixing Bugs
MicroByte Title: Fixing Bugs

Domain: Android App Development

Scope of the learners:
1. Learners will learn how to fetch data from the given URL.
2. Able to learn how to use API calls.

Activities:
1. Layout Imrovement 
2. Coding Phase in MainActivity.java

### PR DESCRIPTION
**Pull request by deepak-prajapatii**
# Closes #88 

Checklist:
- [x] Added activities for learners
- [x] Didn't give the solutions to the added activities straight away
- [x] Followed a LearnByDoing approach
- [x] Clean and well-documented code (if source code files are involved)
- [x] Added images for better understanding if deemed necessary

### CODE REVIEW

**Comment by deepak-prajapatii** [comment]
@amoghrajesh Today is the last day of the hacktberfest. So Review my PR ASAP, if there are errors so i can update my pr within the time.

**Comment by sudhanshutiwari264** [comment]
@deepak-prajapatii we will review this asap !
Till then explore it and find the issues you can raise from this and other mb !
Happy contributing !🎉

**Comment by deepak-prajapatii** [comment]
> @deepak-prajapatii we will review this asap !
> Till then explore it and find the issues you can raise from this and other mb !
> Happy contributing !🎉

Sure

File: Video Player Application/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Introduction
+
+Hello Geeks, We all use Youtube for watching videos for fun, learning etc. In this microbyte, we will going to make the app like youtube but little different. we will just going to make a single activity playing Video.
```
Comment (amoghrajesh) [inline Video Player Application/README.md:3]: we will be going to make an app like YouTube*
The last sentence doesn't make sense to me. Correct it

**Review by amoghrajesh — CHANGES_REQUESTED** [review]
Hey @deepak-prajapatii!
Sorry for the delay. 
I have a few changes to request to you:

1. It is not an hour of content. You need to add a few more activities. (Some contributors have added ~10 of them. Just telling so that you get an idea).
2. The intro is very vague and light. Intro is the part where you sell the MicroByte to the users. So have it strong, so that a reader will feel like moving forward with it. Kindly add a few more lines and make it sound interesting.
Don't write paragraphs, instead make it to the point points.

3. Your activity structure is off. You need to have it be like `Activity1`, `Activity2` ... so on. Not like Activities title with points within. (Have a proper title for all of your activities - for eg: First one could be named as - `Looking at the final product`

4. There are a lot of typos in the file. Please correct them. I will point out a few, rest you need to correct. 

5. The activity intros are not clear. Kindly make it more detailed and clear what a learner will achieve once he reaches there.

6. The alignment of text is quite off. For example, see the markings in screenshot below. This is present in the entire file, correct it.
image

Make these changes asap and we shall re-review it. 

Thanks!

**Comment by deepak-prajapatii** [comment]
> Hey @deepak-prajapatii!
> Sorry for the delay.
> I have a few changes to request to you:
> 
> 1. It is not an hour of content. You need to add a few more activities. (Some contributors have added ~10 of them. Just telling so that you get an idea).
> 2. The intro is very vague and light. Intro is the part where you sell the MicroByte to the users. So have it strong, so that a reader will feel like moving forward with it. Kindly add a few more lines and make it sound interesting.
>    Don't write paragraphs, instead make it to the point points.
> 3. Your activity structure is off. You need to have it be like `Activity1`, `Activity2` ... so on. Not like Activities title with points within. (Have a proper title for all of your activities - for eg: First one could be named as - `Looking at the final product`
> 4. There are a lot of typos in the file. Please correct them. I will point out a few, rest you need to correct.
> 5. The activity intros are not clear. Kindly make it more detailed and clear what a learner will achieve once he reaches there.
> 6. The alignment of text is quite off. For example, see the markings in screenshot below. This is present in the entire file, correct it.
>    image
> 
> Make these changes asap and we shall re-review it.
> 
> Thanks!

Sorry for the inconvenience, I'll update it ASAP.

**Comment by Kevinpaulose05** [comment]
Why is this pattern recurring again? This is also a small and insufficient MB. We won't be accepting such submissions as mentioned earlier. Pls refer to other merged PRs. See what they are trying to do. Ur submissions involves some small activity as an application. There is absolutely very little learning happening here. Let me remind u again that ppl who have made only one PR but have made a quality content have way more weight age than the ones u r proposing. I won't be able to reiterate this process each time. Hiding the code is not all we see. There should be content that helps a person learn some technical concept, not just be happy by building something and not know how all that happened. We would be simply closing such PRs henceforth. Pls reassess and start over.

**Comment by deepak-prajapatii** [comment]
> Why is this pattern recurring again? This is also a small and insufficient MB. We won't be accepting such submissions as mentioned earlier. Pls refer to other merged PRs. See what they are trying to do. Ur submissions involves some small activity as an application. There is absolutely very little learning happening here. Let me remind u again that ppl who have made only one PR but have made a quality content have way more weight age than the ones u r proposing. I won't be able to reiterate this process each time. Hiding the code is not all we see. There should be content that helps a person learn some technical concept, not just be happy by building something and not know how all that happened. We would be simply closing such PRs henceforth. Pls reassess and start over.

Sorry for the inconvenience, I'll update it ASAP.

**Pull request closed without merging**